docs/contracts/inheritance.rst

```diff
@@ -25,21 +25,24 @@ Details are given in the following example.
 
     pragma solidity >=0.5.0 <0.7.0;
 
-    contract owned {
+
+    contract Owned {
         constructor() public { owner = msg.sender; }
-        address payable owner;
+        address payable public owner;
```

Comment:
Why public?

Reply:
If you mark it as private then the other contracts in the example can't access the variable, for example:

```
    function kill() public {
        if (msg.sender == owner) selfdestruct(owner);
    }
```

Reply:
Marking it public adds a getter function which we would not have otherwise.

Reply:
Should be `internal`.

Reply:
But again: Perhaps better without keyword?

Reply:
Please change this.